common-npm-packages/packaging-common/locationUtilities.ts

```diff
@@ -149,7 +154,9 @@ export function getWebApiWithProxy(serviceUri: string, accessToken?: string): vs
     const options: IRequestOptions = {
         proxy: tl.getHttpProxyConfiguration(serviceUri),
         allowRetries: true,
-        maxRetries: 5
+        maxRetries: 5,
+        socketTimeout: webApiOptions && webApiOptions.socketTimeout,
```

Comment:
What values will be used if `webApiOptions` or `socketTimeout` property are not defined?